x-pack/plugins/search_playground/common/types.ts

```diff
@@ -33,6 +33,7 @@ export enum LLMs {
   openai = 'openai',
   openai_azure = 'openai_azure',
   bedrock = 'bedrock',
+  gemini = 'gemini',
```

Comment:
I don't think you are planning on integrating with this application, we should not be modifying this code

Reply:
Agreed - shouldn't be addressed in this PR.

Reply:
done